StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetLPMConfirmFlowTests.swift

```diff
@@ -51,6 +52,8 @@ final class PaymentSheet_LPM_ConfirmFlowTests: XCTestCase {
                 return STPTestingJPPublishableKey
             case .BR:
                 return STPTestingBRPublishableKey
+            case .FR:
+                return STPTestingFRPublishableKey
```

Comment:
We should add a test for alma here, too.

Reply:
Yeah that was the plan but alma test mode currently routes to alma's backend, which is not fully working. You can see from this [payment authorization upstream conversation](https://admin.corp.stripe.com/upstream-conversation/payauthz_QlFkb2l4UENDT0dTOTBJeW41Rk1NTEstU0JrNVJkejMybXd3N285dHVIQSMSFwoVYWNjdF8xSnRnZlFLRzZ2YzdyN1lD). Therefore currently the confirm workflow doesn't really work.